test/instrumentation/modules/mongodb/mongodb.test.js

```diff
@@ -43,6 +43,9 @@ const TEST_DB = 'elasticapm';
 const TEST_COLLECTION = 'test';
 const TEST_USE_CALLBACKS = semver.satisfies(MONGODB_VERSION, '<5');
 
+// From mongodb@3.6.0 and up CommandSuccessEvents may contain errors
+const MONGODB_SUCESS_WITH_ERRORS = semver.satisfies(MONGODB_VERSION, '>=3.6.0');
```

Comment:
nit: typo

```suggestion
const MONGODB_SUCCESS_WITH_ERRORS = semver.satisfies(MONGODB_VERSION, '>=3.6.0');
```

Reply:
At this pace I'll earn a `typo master` badge from github 😅